model-optimizer/mo/utils/unittest/test_data/mxnet_synthetic_gru_bidirectional_FP16_1_v6.xml

```diff
@@ -20,7 +20,7 @@
 				</port>
 			</input>
 			<output>
-				<port id="1">
+				<port id="1" names="abc,def">
```

Comment:
I don't see any check in the test that the tensor names that we read from this file are correct.

Reply:
Ok, I added separate test for this.

And this change was removed. This approach worked too, we compare names inside IE Engine tests, but separate tests will be more clear, yes.